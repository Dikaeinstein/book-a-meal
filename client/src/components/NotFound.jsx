import React from 'react';
import Button from './util/Button';
import history from '../helpers/history';

const NotFound = () => {
  const handleClick = () => {
    history.goBack();
  };
  return (
    <div className="bg-light text-center not-found">
      <h1>404</h1>
      <h2>Sorry!</h2>
      <h2>Page not found :)</h2>
      <Button
        className="btn btn-notfound"
        onClick={handleClick}
        value="Go Back"
      />
    </div>
  );
};

export default NotFound;
