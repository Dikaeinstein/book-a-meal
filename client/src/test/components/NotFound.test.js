import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

describe('<NotFound />', () => {
  it('should render correctly', () => {
    const tree = shallow(<NotFound />);
    expect(tree).toMatchSnapshot();
  });
});
