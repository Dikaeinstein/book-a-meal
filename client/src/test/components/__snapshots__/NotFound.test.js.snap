// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render correctly 1`] = `
<main>
  <div
    className="bg-dark-light text-center not-found"
    style={
      Object {
        "minHeight": "calc(100vh - 151px)",
      }
    }
  >
    <h1>
      404
    </h1>
    <h2>
      Sorry!
    </h2>
    <h2>
      Page not found :)
    </h2>
    <br />
    <br />
    <br />
    <Button
      className="btn btn-notfound"
      onClick={[Function]}
      value="Go Back"
    />
  </div>
  <Footer />
</main>
`;
